index.ts

```diff
@@ -31,12 +34,12 @@ app.engine(
   "hbs",
   handlebars({
     extname: "hbs",
-    defaultLayout: false,
+    defaultLayout: "",
   })
 );
 
-function getPropertyByPath(obj, path) {
-  return path.split('.').reduce((data, key) => {
+function getPropertyByPath(obj: object, path: string) {
+  return path.split(".").reduce((data, key) => {
```

Comment:
Why is changing quotes here a _necessary_ change?

Reply:
Wasn't a necessary change on my side. Must have been husky.